molecule/default/molecule.yml

```diff
@@ -83,6 +83,7 @@ provisioner:
         zookeeper_id: 2
         kafka_broker_id: 1
         kafka_listener_hostname: server-2
+        kafka_download_validate_certs: no
```

Comment:
Don't validate the certs for this CentOS 7 image as it's version of OpenSSL and cacerts means it encounters the Lets Encrypt cert expiry issue.